app/(auth)/layout.tsx

```
@@ -0,0 +1,60 @@
+        <div className="z-10 relative lg:mt-4 lg:mb-16">
+          <blockquote className="auth-blockquote">
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Neque vitae
+            non aliquid nostrum eveniet ullam doloremque consequuntur vero ipsa.
+          </blockquote>
+          <div className="flex items-center justify-between">
+            <div>
+              <cite className="auth-testimonial-author">- TOGB R.</cite>
+              <p className="max-md:text-xs text-gray-500">Analyst</p>
+            </div>
+            <div className="flex items-center gap-0.5">
+              {[1, 2, , 3, 4, 5].map((star) => (
```

Biome (2.1.2)

[error] This array contains an empty slots..

The presences of empty slots may cause incorrect information and might be a typo.
Unsafe fix: Replace hole with undefined


(lint/suspicious/noSparseArray)